### PR TITLE
glibc: Building glibc from sources

### PR DESCRIPTION
**Pull request by ijaved7**
Signed-off-by: ijaved <iqra_javed@mentor.com>

### CODE REVIEW

**Comment by Noor-Ahsan** [comment]
@ijaved7 you have commited gedit temp files as well. see recipes-sourcery/glibc-sourcery/glibc-sourcery.bb~. Kindly clean this up.

**Comment by ijaved7** [comment]
@Noor-Ahsan 
Okay, i'll remove it.

**Comment by ijaved7** [comment]
@Noor-Ahsan  
It's deleted.

File: recipes-sourcery/glibc-sourcery/glibc-sourcery.bb [hunk 1]
```diff
@@ -95,23 +98,24 @@ linux_include_subdirs = "asm asm-generic bits drm linux mtd rdma sound sys video
 do_install_append () {
     for dir in ${linux_include_subdirs}; do
         rm -rf "${D}${includedir}/$dir"
+        echo "ahsan"
```
Comment (Noor-Ahsan) [inline recipes-sourcery/glibc-sourcery/glibc-sourcery.bb]: this needs to be removed as well

**Comment by ijaved7** [comment]
@Noor-Ahsan  
Updated.

**Pull request merged**